src/julia.expmap.in

```diff
@@ -13,30 +13,26 @@
     rec_backtrace;
     julia_*;
     libsupport_init;
-    localtime_r;
     memhash;
     memhash32;
     memhash32_seed;
     memhash_seed;
     restore_signals;
     u8_*;
     uv_*;
-    add_library_mapping;
     utf8proc_*;
     jlbacktrace;
     jlbacktracet;
-    _IO_stdin_used;
     _Z24jl_coverage_data_pointerN4llvm9StringRefEi;
     _Z22jl_coverage_alloc_lineN4llvm9StringRefEi;
     _Z22jl_malloc_data_pointerN4llvm9StringRefEi;
     _jl_timing_*;
     LLVMExtra*;
     JLJIT*;
-    llvmGetPassPluginInfo;
 
     /* freebsd */
-    environ;
-    __progname;
+    environ*;
+    __progname*;
```

Comment:
Why the new `*` here?

Reply:
I can't figure out how to have `environ` and `progname` be defined. The *, means lld won't throw an undefined error.